lib/python/vdm/server/HTTPListener.py

```diff
@@ -499,6 +499,127 @@ def get_volt_jar_dir():
     return os.path.realpath(os.path.join(Global.MODULE_PATH, '../../../..', 'voltdb'))
 
 
+def get_port(ip_with_port):
```

Comment:
Do we validate that the port specified is a number? May be we can do that also here?